systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationIsolatedST.java

```diff
@@ -652,6 +652,8 @@ void testKeycloakAuthorizerToDelegateToSimpleAuthorizer(ExtensionContext extensi
 
     @BeforeAll
     void setUp(ExtensionContext extensionContext)  {
+        clusterOperator.unInstall();
+        clusterOperator.defaultInstallation().createInstallation().runInstallation();
```

Comment:
Why don't you add it to `setupCoAndKeycloak` method? Currently it's only deploying Keycloak. (so it has wrong name or CO installation should be inside it)

Reply:
Yes, make sense 👌